ear-production-suite-plugins/plugins/scene/src/scene_frontend_connector.cpp

```diff
@@ -727,9 +727,9 @@ void JuceSceneFrontendConnector::removeTabClicked(
     progCount = p_->getProgrammeStore()->programme_size();
   }
   if (progCount == 1) {
-    AlertWindow::showNativeDialogBox(
+    NativeMessageBox::showMessageBoxAsync(MessageBoxIconType::NoIcon,
         String("Cannot delete last programme"),
-        "The Scene must always have at least one programme.", false);
+        "The Scene must always have at least one programme.", nullptr);
```

Comment:
Where does the box pop up if the associated component is nullptr? If it stays within the plug-in window then no problem, but if it could be anywhere, it might be missed by the user.

Reply:
Well anecdotally, it pops up over the plugin window. But yes, good point. I'll go check the docs :)